graspy/plot/plot.py

```diff
@@ -404,7 +404,9 @@ def gridplot(
         elements.
     palette : str, dict, optional, default: 'Set1'
         Set of colors for mapping the ``hue`` variable. If a dict, keys should
-        be values in the hue variable
+        be values in the hue variable.
+        For acceptable string arguments, See the palette options in 
+        :matplotlib:`matplotlib<colormaps>`
```

Comment:
this will also need to be changed

Reply:
see this https://matplotlib.org/devel/documenting_mpl.html#referring-to-other-documents-and-sections
the way to reference using intersphinx should be similar

Reply:
or it may be more like this https://matplotlib.org/devel/documenting_mpl.html#referring-to-other-code

Reply:
try doing (from the command line) 
`pip install sphinx`
`python -m sphinx.ext.intersphinx 'https://matplotlib.org/objects.inv'`
this will print out a list of objects you can reference using intersphinx, then you'll just need to find the right one for that colormapping page

Reply:
Hi @bdpedigo , I got why it didn't work. It should be:
`python -msphinx.ext.intersphinx https://matplotlib.org/objects.inv`
Now it works. I think I find the page, but I don't know how to reference it...
![image](https://user-images.githubusercontent.com/69659070/94178433-acdae980-fecd-11ea-87aa-8016b35ffab4.png)
It's under std:doc